spark-jobs/pom.xml

```diff
@@ -191,6 +190,20 @@
                             <exclude>net.jpountz.lz4:*</exclude>
                         </excludes>
                     </artifactSet>
+                    <relocations>
+                        <!-- This was introduced in #86 Set up Standardization and Conformance Menas auth.
+                             If not shaded the following exception will be thrown:
+                               Caused by: java.lang.ClassNotFoundException: za.co.absa.atum.model.RunState
+	                           at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
+	                           ...
+	                           at com.fasterxml.jackson.module.scala.deser.EnumerationDeserializer.deserialize(EnumerationDeserializerModule.scala:40)
+                               java.lang.NoClassDefFoundError: com/fasterxml/jackson/datatype/jsr310/JavaTimeModule
+                        -->
+                        <relocation>
+                            <pattern>com.fasterxml.jackson</pattern>
+                            <shadedPattern>com.shaded.fasterxml.jackson</shadedPattern>
```

Comment:
Maybe it does not matter here (moreover, if the purpose for relocation is only not to interfere), but usually, I find a good practice to prefix the newyl relocated package name by our own namespace (e.g. `za.co.absa.shaded.com.fasterxml.jackson`). This also has the benefit of minimizing the chance that another jar chose the same shaded location.

Reply:
Good point! Will do